CVS-46606 Custom node implementation for OCR (#541)
openvinotoolkit/model_server: src/custom_nodes/east_ocr/README.md

# Custom node for OCR implementation with east-resnet50 and crnn models 

This custom node analyses the response of east-resnet50 model. Based on the inference results and the original image,
it generates a list of detected boxes for text recognition. 
Each image in the output will be resized to the predefined target size to fit the next inference model in the 
DAG pipeline.

> Beside the set of detected text boxes, in the second output are returned their labels with information about geometry

Reviewer: ```suggestion
Additionally to the detected text boxes, in the second output are returned their labels with information about geometry
```

and scores but only for the filtered list of detections.  

# Building custom node library

> You can build the shared library of the custom node simply by command in this custom node folder context:

Reviewer: ```suggestion
You can build the shared library of the custom node simply by running command in this custom node folder context:
```

Reviewer: Point to the exact directory instead of "folder context"
